conf/messages.cy

```diff
@@ -1341,8 +1339,8 @@ disposalPrice.trust.Gifted.error.required=Nodwch y gwerth marchnadol pan wnaeth
 disposalPrice.trust.indirect.Gifted.error.required=Nodwch werth marchnadol y cyfranddaliadau pan gafodd eu gwerthu neu eu gwaredu
 disposalPrice.personalRep.Gifted.error.required=Nodwch werth marchnadol yr eiddo pan gafodd ei waredu
 disposalPrice.personalRep.indirect.Gifted.error.required=Nodwch werth marchnadol y cyfranddaliadau pan gafodd eu gwerthu neu eu gwaredu
-disposalPrice.personalRepInPeriodOfAdmin.Gifted.error.required=Nodwch enillion y gwarediad
-disposalPrice.personalRepInPeriodOfAdmin.indirect.Gifted.error.required=Nodwch enillion y gwarediad
+disposalPrice.personalRepInPeriodOfAdmin.Gifted.error.requiredNodwch werth marchnadol yr eiddo pan gafodd ei waredu
```

Comment:
looks like you're missing out an `=` after `required`

Reply:
thanks, will fix it